src/coreclr/nativeaot/Runtime/GCMemoryHelpers.cpp

```diff
@@ -36,15 +36,18 @@ COOP_PINVOKE_CDECL_HELPER(void *, RhpGcSafeZeroMemory, (void * mem, size_t size)
 
 COOP_PINVOKE_HELPER(void, RhBulkMoveWithWriteBarrier, (uint8_t* pDest, uint8_t* pSrc, size_t cbDest))
 {
+#if TARGET_ARM64
```

Comment:
Can we define `GCHeapMemoryBarrier` macro that matches what we do in CoreCLR?

https://github.com/dotnet/runtime/blob/e18e4aa3b3a914a054752261d136ace6b8bcdd87/src/coreclr/vm/common.h#L338-L352